tests/test_fhir.py

```diff
@@ -60,6 +60,27 @@ def redis_session(client, redis_handle):
     redis_handle.set(session_key, pickle.dumps(session_data))
 
 
+@fixture
+def auth_extensions(client, redis_handle):
+    """Returns a set of extensions typically used for auth, a subset of a FHIR conformance statement"""
+    fhir_auth_extensions = [
+        {
+            "url": "token",
+            "valueUri": "https://cpsapisandbox.virenceaz.com:9443/demoAPIServer/oauth2/token"
+        },
+        {
+            "url": "authorize",
+            "valueUri": "https://cpsapisandbox.virenceaz.com:9443/demoAPIServer/oauth2/authorize"
+        },
+        {
+            "url": "register",
+            "valueUri": "https://cpsapisandbox.virenceaz.com:9443/demoAPIServer/oauth2/registration"
+        }
+    ]
+
+    return fhir_auth_extensions
```

Comment:
minor nit - but no point in creating a named variable just to return it - why not simply:
return [all of the above]